core/web/loader/job_spec_errors.go

```diff
@@ -0,0 +1,60 @@
+package loader
+
+import (
+	"context"
+
+	"github.com/graph-gophers/dataloader"
+
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/utils/stringutils"
+)
+
+type jobSpecErrorsBatcher struct {
+	app chainlink.Application
+}
+
+func (b *jobSpecErrorsBatcher) loadByJobIDs(_ context.Context, keys dataloader.Keys) []*dataloader.Result {
+	// Create a map for remembering the order of keys passed in
+	keyOrder := make(map[string]int, len(keys))
+	// Collect the keys to search for
+	var jobIDs []int32
+	for ix, key := range keys {
+		id, err := stringutils.ToInt32(key.String())
+		if err == nil {
+			jobIDs = append(jobIDs, id)
+		}
+
+		keyOrder[key.String()] = ix
+	}
+
+	specErrors, err := b.app.JobORM().FindSpecErrorsByJobIDs(jobIDs)
+	if err != nil {
+		return []*dataloader.Result{{Data: nil, Error: err}}
+	}
+
+	// Generate a map of spec errors to jobIDs
```

Comment:
Looks like this should be "a map of jobIDs to spec errors"

Reply:
Addressed this comment